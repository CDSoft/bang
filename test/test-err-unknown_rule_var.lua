generator(false)

rule "doit" {
    description = "...",
    command = "...",
    depfile = "...",
    deps = "...",
    dyndep = "...",
    pool = "...",
    msvc_deps_prefix = "...",
    generator = "...",
    restat = "...",
    rspfile = "...",
    rspfile_content = "...",
    foo = "this is not a rule variable", -- [test/test-err-unknown_rule_var.lua:3] ERROR: rule doit: unknown variables: foo
}
