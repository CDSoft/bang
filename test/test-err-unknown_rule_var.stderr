[test/test-err-unknown_rule_var.lua:3] ERROR: rule doit: unknown variables: foo
